### ISSUE TITLE
Document Daily Progress & New Tasks -- Casey

### ISSUE THREAD

**Issue by Ameobea** [Ameobea/network_research#22]
This is set as a daily task for each student Team Member.
Update the Asana project tasks, and research record.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/118867524991974/137350695795083)

**Comment by Ameobea** [comment]
https://docs.google.com/document/d/1hi7s4wPpUsjLaunYAVvxUs-RDC8aGzNZCpgi3B3U5As/edit

**Comment by Ameobea** [comment]
Today was focused around creating scripts for analyzing correlations between different network attributes.  In 69aa524d0404fb0ce36813edaaad014b638a758b, f44241e7f401cb977d72948f36267ddeba4af4df, and a5a604e31dbd57c8a0867ec04903c98f23bdfbc0, I added in the `correlate.py` and `plotCorrelations.py`.  These have the end result of generating scatter plots containing correlation data between different attributes within networks as detailed in the above linked google docs document and #27 .  

I will eventually use this correlation data to identify useless attribute correlations (those that are always correlated regardless of the type of network) as opposed to useful correlations (those that only exist in a certain type of network).  _My goal is to use polynomial fits, linear regressions, or some other kind of statistical analysis method to identify baselines for all networks and certain types of networks with which to compare unknown networks as a means of classification._

In addition to that coding work, I also downloaded many (10GB+) of network files, mostly from SNAP, for use in generating baseline correlation statistics.  I spent some time looking into different statistical methods that I could use to find correlations in the network data, reading about correlograms and eventually settling on simple scatter plots.  I explored the possibility of using R for the plot generation (I'd used it for that in the past), but then I remembered about matplotlib's PyPlot that worked perfectly in native Python.
